robottelo/ui/contentviews.py

```diff
@@ -481,16 +481,21 @@ def add_remove_package_groups_to_filter(self, cv_name, filter_name,
             self.click(locators['contentviews.remove_pkg_group'])
 
     def add_remove_errata_to_filter(self, cv_name, filter_name,
-                                    errata_ids, is_add=True):
+                                    errata_ids=None, select_all=None,
+                                    is_add=True):
         """Add/Remove errata to/from selected filter for inclusion/exclusion"""
         self.go_to_filter_page(cv_name, filter_name)
         if is_add:
             self.click(tab_locators['contentviews.tab_add'])
         else:
             self.click(tab_locators['contentviews.tab_remove'])
-        for errata_id in errata_ids:
-            self.click(
-                locators['contentviews.select_errata_checkbox'] % errata_id)
+        if errata_ids is not None:
+            for errata_id in errata_ids:
+                self.click(
+                    locators.contentviews.select_errata_checkbox % errata_id)
+        if select_all is not None:
```

Comment:
logically, it is better to separate selection procedure and actual action flow, but probably we cannot do that in current implemenation